src/node-utils.ts

```diff
@@ -406,37 +407,56 @@ function hasStaticModifierFlag(node: ts.Node): boolean {
 
 /**
  * Finds the next token based on the previous one and its parent
- * @param {ts.Token} previousToken The previous ts.Token
- * @param {ts.Node} parent The parent ts.Node
+ * Had to copy this from TS instead of using TS's version because theirs doesn't pass the ast to getChildren
+ * @param {ts.Token} previousToken The previous TSToken
+ * @param {ts.Node} parent The parent TSNode
+ * @param {ts.SourceFileLike} ast The TS AST
```

Comment:
What is the difference between SourceFileLike and SourceFile? The comment doesn't match the usage

Reply:
My mistake, fixed to `SourceFile`.